docs/docs/install.md

```diff
@@ -10,7 +10,7 @@ You'll want to make sure your local environment is ready by installing the follo
 
 ### Docker
 
-If you don't have them installed yet, install [Docker] and [docker-compose].
+If you don't have them installed yet, install [Docker] and [Compose].
```

Comment:
Should we mention the minimum required versions here?

Reply:
Great idea. I don't know what the minimum should be.